internal/summary/summary.go

```diff
@@ -24,29 +33,87 @@ func NewSummary() *Summary {
 }
 
 // Gather collates data into Summary
-func (s *Summary) Gather(services []model.Service) {
+func (s *Summary) Gather(services []model.Service) error {
 
 	for _, svc := range services {
-		var path string
 		// Assume that, in practice, there is only one mount
+		path := ""
 		if len(svc.Mounts) > 0 {
 			path = svc.Mounts[0].Repo.LocalPath
-		} else {
-			path = ""
 		}
-		s.Services = append(s.Services, &Service{
-			Name: string(svc.Name),
-			Path: path,
-		})
+
+		svcSummary := &Service{
+			Name:    string(svc.Name),
+			Path:    path,
+			k8sData: []k8sDatum{},
```

Comment:
don't need this line -- leave it out and the field will auto-initialize to the empty value ([ex here](https://play.golang.com/p/-tdSejvykOi)).